Assets/MixedRealityToolkit/Inspectors/MixedRealityStandardShaderGUI.cs

```diff
@@ -1018,5 +1063,36 @@ protected static void SetColorProperty(Material material, string propertyName, C
                 material.SetColor(propertyName, propertyValue.Value);
             }
         }
+
+        [MenuItem("Mixed Reality Toolkit/Utilities/Upgrade MRTK Standard Shader for Lightweight Render Pipeline")]
+        protected static void UpgradeShaderForLightweightRenderPipeline()
+        {
```

Comment:
Odds are people will be clicking this just to see what happens.

If they do this in a project that isn't using the lightweight render pipeline, their shaders will stop working and render as pink, correct?

We should confirm their choice with a yes/cancel dialog, and we should add a menu operation that reverses the procedure.

Reply:
That's a good idea! I'll add that.

Reply:
You could also confirm they have the LWRP dependency before upgrading.

Reply:
@Jarodshow Great idea! I just added a menu item validator to make sure they at minimum have a renderPipelineAsset specified in their project settings. (Just validating any SRP asset in case a developer has a custom RP based on the LWRP.)